src/scripts/pbs_status.py

```diff
@@ -232,13 +232,13 @@ def qstat(jobid=""):
 
     starttime = time.time()
     log("Starting qstat.")
-    if re.search(r'PBSPro', qstat_version):
-        command = (qstat_bin, '-f', jobid) # -1 conflicts with -f in PBS Pro
-    else:
-        command = (qstat_bin, '-f', '-1', jobid)
+    command = (qstat_bin, '-f')
```

Comment:
I'm a much bigger fan of a list, but that's just a personal preference.  I was wondering if the popen command only took tuples, which doesn't really make sense the more I think of it.

If this all works for you, no reason to hold up the pull request on something silly for me.